operations/network/network_management_install_guide/aruba/mclag_link_ha.md

```diff
@@ -0,0 +1,15 @@
+# VSX: MCLAG link HA
+
+The intent here is to show case a typical MCLAG link failover scenario where one of the links to the edge device, whether that is connected switch or server. 
+ 
+In below a typical traffic pattern coming off from MCLAG connected device, as you can see the traffic is going north to south and ISL is not carrying any traffic. The only time ISL would carry traffic would be if one of the links to downstream devices would be down. 
+
+![](../docs-csm/operations/network/network_management_install_guide/img/mclag_link_ha.png) 
```

Comment:
Not able to see picture.

Reply:
I dont think the path is working yet since it is not pushed to repo. I'll see if I can fix it.